### ISSUE TITLE
Load cards in approximate rows instead of columns

### ISSUE THREAD

**Issue by tibonova** [feruzm/esteem-desktop#3]
When I use the app in big window and there are multiple columns, the cards are shorted by date and time, 1 column filled 1st to the bottom, then 2nd column to the bottom, and so on. It would be better if we could see posts on the screen that are closer to each other in time (or in other criteria.) 

The implementation of this would be easy in _compact view_.
In Card view, the better approximation of the placements of cards would require that the app have to give back the card's height data after it's been loaded. With that data, long or short cards won't be the breaker of this order. 

TBH, it annoys me so much I'll test the app only with 1 big column. (I also like using apps like this in that way more, so it's not a big issue for me. I cannot speak for others, though.)

**Comment by feruzm** [comment]
yes, I am thinking about having slightly different layout other than columns. Yes, card height is different depending on images, although perhaps it would be wise to pick on size and try to fit image into it and nice grid style view.
